packages/web-shared/components/ContentSingle.js

```diff
@@ -255,6 +257,54 @@ function ContentSingle(props = {}) {
           </Box>
         ) : null}
         {/* Display content for sermons */}
+        {hasParent ? (
+          <>
+            <H3 flex="1" mr="xs" mb="xs">
+              This Series
+            </H3>
+            <Box flex="1" mb="l" minWidth="180px" {...props}>
```

Comment:
I assume there's not existing card components?

Reply:
There's not, and there was enough difference between this card and the hero card that I punted on trying to DRY it up